docs/source/plugins.rst

```diff
@@ -311,3 +312,14 @@ The following known plugins provide support for running `MLflow projects <https:
 against custom execution backends.
 
 - `mlflow-yarn <https://github.com/criteo/mlflow-yarn>`_ Running mlflow on Hadoop/YARN
+
+Tracking Store Plugins
```

Comment:
@karthik-77 Is this part of upstream changes? What are we adding details for Tracking store plugins?

Reply:
Yes it is a upstream change. I took this file from upstream master branch